### ISSUE TITLE
Tests crash/skip for php-7.4 with opcache config

### ISSUE THREAD

**Issue by lavturo** [php/pftt2#89]
I was testing php-7.4.1RC1 and saw that a lot more tests were classified as CRASH or SKIP than usual. 

When pftt is **not** configured with opcache, there are:
 ~11,000 tests passed, ~600 tests skipped, ~2 tests crashed. 

When pftt is configured with opcache, there are: 
~15 tests passed, ~4,600 tests skipped, ~7,400 tests crashed. 

Seems like this issue might have been around for quite for a while. Previously, I have been running an older version of pftt2 on VMs and this issue did not occur. I recently updated it and now here it is. Will have to look through PRs to see which one might have caused this.

**Comment by lavturo** [comment]
Seems to be for any build, not just php-7.4.

**Comment by cmb69** [comment]
> Seems to be for any build, not just php-7.4.

These are actually good news (otherwise we might have had a bad build).

Are you sure the crashes are caused by PFTT changes? Could as well be something different. Maybe try to run some of these crashing test cases with run-test.php, or even with plain PHP.

**Comment by lavturo** [comment]
I am almost certain the crashes are caused by pftt changes. I still have a backup copy of pftt2 on the VMs and when I run the tests on them, they get tested normally (not all as crash/skip). However, I am a bit unsure which version of pftt (how many commits it was behind) the ones on the VM are. 

Just in case, using run-test.php, I did try to check some tests that resulted as a crash and they pass.

**Comment by lavturo** [comment]
Strange. I was able to replicate the issue locally yesterday, even after a clean build. However, I could not replicate the bug this morning. I exported this to the VMs and they work perfectly fine now.

However, I was able to replicate this bug again after cleaning and building pftt2 after switching my project back to what I was working on (issue 5). The tests would crash/skip again. Here, I realized if I restart my PC, it would work again.
